api/v1alpha1/seccompprofile_types.go

```diff
@@ -96,6 +96,8 @@ type SeccompProfileStatus struct {
 	Path            string   `json:"path,omitempty"`
 	Status          string   `json:"status,omitempty"`
 	ActiveWorkloads []string `json:"activeWorkloads,omitempty"`
+	// The path the profile to provide in a pod spec
+	SeccompProfileLocalhostProfile string `json:"seccompProfile.localhostProfile,omitempty"`
```

Comment:
What about:
```suggestion
	LocalhostProfile string `json:"localhostProfile,omitempty"`
```
And then we enhance the description to mention that is is the container or pod `securityContext.seccompProfile.localhostProfile` path.

Reply:
done